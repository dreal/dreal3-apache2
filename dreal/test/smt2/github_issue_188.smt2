;; From https://github.com/dreal/dreal4/issues/188
(set-logic QF_NRA)
(set-option :worklist-fixpoint true)
(declare-const v1 Bool)
(declare-const v2 Bool)
(declare-const v3 Bool)
(declare-const v4 Bool)
(declare-const v6 Bool)
(declare-const v7 Bool)
(declare-const v8 Bool)
(declare-const v9 Bool)
(declare-const v11 Bool)
(declare-const r0 Real)
(declare-const r2 Real)
(declare-const r4 Real)
(declare-const r5 Real)
(declare-const r7 Real)
(declare-const r8 Real)
(declare-const r13 Real)
(declare-const r14 Real)
(declare-const r15 Real)
(push 1)
(assert false)
(check-sat)
(pop 1)
(declare-const v24 Bool)
(declare-const r19 Real)
(declare-const r20 Real)
(assert (<= r0 r4))
(check-sat)
(declare-const v30 Bool)
(declare-const v31 Bool)
(declare-const v33 Bool)
(declare-const v34 Bool)
(check-sat)
(declare-const r28 Real)
(declare-const r29 Real)
(declare-const v44 Bool)
(declare-const r30 Real)
(check-sat)
(declare-const v45 Bool)
(check-sat)
(declare-const v46 Bool)
(declare-const v48 Bool)
(declare-const r35 Real)
(check-sat)
(assert (>= r30 r35))
(check-sat)
