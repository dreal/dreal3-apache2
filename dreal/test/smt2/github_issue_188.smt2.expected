unsat
delta-sat with delta = 0.001
delta-sat with delta = 0.001
delta-sat with delta = 0.001
delta-sat with delta = 0.001
delta-sat with delta = 0.001
delta-sat with delta = 0.001
